client/components/Landing/LandingStats/LandingStats.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  StatsCardLayout1,
+  StatsCardLayout2,
+  StatsCardLayout3,
+  StatsCardLayout4,
+  StatsCardLayout5,
+  StatsCardLayout6,
+} from '~/components/PlatformStats/v2/StatsCards';
+
+export default {
+  title: 'StatsCards',
+};
+
+export const StatsCard1 = () => (
+  <StatsCardLayout1
```

Comment:
Now I see what you meant regarding the styles. It seems the cards have certain styles applied from the parent (`LandingStats`).

This is bad design and I should have caught it when we reviewed them. Not a big deal then, we're gonna fix here.

So, we remove all the card-related styles  from `LandingStats.module.scss`. It should only contain positioning styles for the cards. Thus these should be moved to Background 👇
1. the `padding`
2. the `before`

This way the cards in Storybook will look like the ones in our main app, apart from the positioning.